app/models/location.rb

```diff
@@ -26,6 +26,8 @@ class Location < ActiveRecord::Base
 
   scope :regional, ->(ids) { where(region_id: ids) }
 
+  scope :active, -> { where(active: true) }
```

Comment:
It's not necessary and I can remove.  My thought was starting to use an explicit `active` scope so eventually the default scope can be removed.